docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Roxie_Overview2.xml

```diff
@@ -1,313 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Definitely looks like a deprecated doc, but perhaps you should get Jim Defabia to approve the PR for completeness.

Reply:
Spoke to Jim about removing this file. Decided to do it as part of this PR instead of opening a separate Jira. He was fine with it.